src/cms/templates/_base.html

```diff
@@ -18,8 +18,8 @@
     {% endif %}
     <a href="{% url 'wiki_redirect' %}" target="_blank" rel="noopener noreferrer"
        class="relative px-2 pt-4 text-gray-800 hover:bg-gray-200 border-r border-gray-400">
-        <i data-feather="help-circle"></i>
-        {% trans 'Help' %}
+        <img class="inline-block" fill="none" width="21" height="21" src="{% static 'images/integreat-icon-contur-only.png' %}">
+        {% trans 'Wiki / Documentation' %}
```

Comment:
I would name it `Documentation (Wiki)`, because the important part here is the `Documentation` and only some may think of our Documentation as "The Wiki".

Reply:
I talked to Daniel about this and he came up with this wording. But as this will only take like 1 minute to change, I would suggest to leave it as Wiki / Documentation for now. Maybe he can take the discussion into the cities-team :)

Reply:
Well, "Documentation" is the standing term for Documentation ;-) We're always discussing "intuition", this is one of these moments. Only a small minority of web users knows what the term wiki actually refers to.

Reply:
Yeah, but there seem to be a time, when we referred to our Documentation as the "Integreat Wiki". So we are at this point again where we have to value between Habit and Conformity :D